### PR TITLE
Add pop-count exercise

### LINKED ISSUE #375: Add `pop-count`
This practice exercise was just added earlier this week. I’d like to port it to Gleam. https://github.com/exercism/problem-specifications/blob/main/exercises/pop-count/canonical-data.json

### PR DESCRIPTION
**Pull request by BNAndras**
Closes #375

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -560,8 +560,7 @@
         "name": "Forth",
         "uuid": "8176ac78-d4fd-475f-ac8b-e205ca7b3a77",
         "difficulty": 9,
-        "practices": [
-        ],
+        "practices": [],
```
Comment (lpil) [inline config.json:563]: Could you revert these unrelated changes please

File: exercises/practice/pop-count/.meta/example.gleam [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+pub fn egg_count(number: Int) -> Int {
+  recurse(number, 0)
+}
+
+fn recurse(number: Int, eggs: Int) -> Int {
+  case number {
+    0 -> eggs
+    _ -> {
+      let new_number = result.unwrap(int.floor_divide(number, 2), 0)
```
Comment (lpil) [inline exercises/practice/pop-count/.meta/example.gleam:12]: This could be written like this:
```suggestion
      let new_number = number / 2
```

**Review by lpil — COMMENTED** [review]
Thank you! I've left two small notes inline

File: config.json (hunk 1 above)
Comment (BNAndras) [reply config.json:563]: I think that snuck in from #381. I’ll take a closer look.

**Comment by BNAndras** [comment]
Per exercism/problem-specifications#2335, the exercise title has bee updated from "Pop Count" to "Eliud's Eggs".

**Review by lpil — APPROVED** [review]
Thank you so much!!
